toolchain/templates/include/helpers.mako

```diff
@@ -64,13 +64,13 @@ fi
 
 cd "${os.path.dirname(input)}"
 
-t_${target.name}_start=$(date +%s)
+t_${target.name}_start=$(date +%s.%N)
 </%def>
 
 <%def name="run_epilogue(target)">
 code=$?
 
-t_${target.name}_stop=$(date +%s)
+t_${target.name}_stop=$(date +%s.%N)
```

Comment:
Note that on Mac:

```console
lawn-10-91-109-237: /Users/spencer $ date +%s.%N
1715972166.N
lawn-10-91-109-237: /Users/spencer $ gdate +%s.%N
1715972183.446123000
```

So, benchmarking will not work on MacOS unless `gdate` (gnu date) is installed and aliased to `date`.